gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -92,6 +92,9 @@
   /** Default setting for grpc message timeout check interval (in milliseconds) */
   public static final long GRPC_MESSAGE_TIMEOUT_CHECK_INTERVAL = 1000;
 
+  /** Default setting for publishing open-census metrics */
+  public static final boolean METRICS_ENABLED = false;
```

Comment:
Probably this should be more specific about where/how metrics are published

Reply:
Along the lines of configuration, couldn't tie it to grpc/cloudmonitoring. Open to suggestions